alf/algorithms/oac_algorithm_test.py

```diff
@@ -90,8 +87,8 @@ def test_oac_algorithm(self, explore=True):
             use_entropy_reward=reward_dim == 1,
             env=env,
             config=config,
-            explore=explore,
-            explore_delta=2.,
+            explore=True,
```

Comment:
explore=explore

Reply:
I just removed the explore=False from the test to save some time, since it is the same as sac. Should I add it back?

Reply:
I see. Didn't notice that.